.changeset/slow-vans-help.md

```diff
@@ -0,0 +1,9 @@
+---
+"@fluid-tools/api-markdown-documenter": minor
```

Comment:
Note that this changeset won't do anything because this package isn't part of the client release group.